core/sdk-core/src/main/java/software/amazon/awssdk/core/RequestOverrideConfiguration.java

```diff
@@ -73,6 +79,43 @@ protected RequestOverrideConfiguration(Builder<?> builder) {
         return apiNames;
     }
 
+    /**
+     * The amount of time to wait for the http request to complete before giving up and timing out. A positive value
+     * enables this feature.
```

Comment:
Should it be "this value will always be positive, if present"? (same comment below)